### PR TITLE
feat: add metrics script for pulling and publishing GitHub issue metrics

### PR DESCRIPTION
**Pull request by childish-sambino**

### CODE REVIEW

File: examples/metrics.py [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+ISSUE_TYPES = {'question', 'support', 'bug', 'enhancement', 'non-library', 'docs', 'security'}
+ISSUE_STATUSES = {'duplicate', 'invalid'}
+
+GOOGLE_SHEET_ID = '1cQOOT5aYxfXOSwEV0cJyf01KkV-uKCBJnKK3PHjouCE'
+
+
+def base_type():
+    return {'nodes': defaultdict(base_type), 'metrics': {}}
```
Comment (thinkingserious) [inline examples/metrics.py]: What is `base_type`?

File: examples/metrics.py [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+
+
+class MetricCollector:
```
Comment (thinkingserious) [inline examples/metrics.py]: Nice!

Thoughts on moving this to its own service (e.g. `metrics`) or add to the `github` service?

I think its beyond the scope of this work, but if you agree, let's make an issue.

File: examples/metrics.py (hunk 1 above)
Comment (childish-sambino) [reply examples/metrics.py]: The name of this function.

File: examples/metrics.py (hunk 2 above)
Comment (childish-sambino) [reply examples/metrics.py]: Meh, I don't see the need to move it to the service.

**Comment by thinkingserious** [comment]
Please update the README.md with usage instructions. Thank you!

**Comment by thinkingserious** [comment]
Hello @childish-sambino,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**